### PR TITLE
allow configurable log level for registry service

### PR DESCRIPTION
**Pull request by pgvishnuram**
## Description

currently loglevel is default to info for registry, in case of override we need to directly update the configmap. This change avoids manual configuration and allow customers/users to override loglevel.

## Related Issues

https://github.com/astronomer/issues/issues/5191

## Testing

with defaults logLevel info

image

with logLevel set to debug
image

## Merging

cherry-pick to release-0.31.

### CODE REVIEW

**Review by andriisoldatenko — APPROVED** [review]

File: tests/chart_tests/test_registry_configmap.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+from tests import supported_k8s_versions
+import yaml
+
+
+@pytest.mark.parametrize(
+    "kube_version",
+    supported_k8s_versions,
+)
+class Test_Registry_Configmap:
+    def test_registry_configmap_with_s3Enabled(self, kube_version):
```
Comment (danielhoherd) [inline tests/chart_tests/test_registry_configmap.py:12]:
```suggestion
    def test_registry_configmap_with_s3_enabled(self, kube_version):
```

**Review by danielhoherd — CHANGES_REQUESTED** [review]
Looks awesome, I love the test coverage! Need to finish filling out the PR description and we'll be good to go.

**Comment by pgvishnuram** [comment]
@danielhoherd - added relavant info for the PR

**Comment by danielhoherd** [comment]
@pgvishnuram still needs merge details.

**Pull request merged**